packages/@uppy/provider-views/package.json

```diff
@@ -21,12 +21,14 @@
   "dependencies": {
     "@uppy/utils": "workspace:^",
     "classnames": "^2.2.6",
-    "nanoid": "^5.0.0",
     "p-queue": "^8.0.0",
     "preact": "^10.5.13"
   },
   "peerDependencies": {
     "@uppy/core": "workspace:^"
   },
-  "stableVersion": "3.11.0"
+  "stableVersion": "3.11.0",
+  "devDependencies": {
+    "vitest": "^1.6.0"
+  }
```

Comment:
Can we move this in between `dependencies` and `peerDependencies` to avoid conflicts with `stableVersion`?

Reply:
Ok, done